app/views/shared/_footer.html.erb

```diff
@@ -15,6 +15,7 @@
 					<div class="table_cell large_padding_right">
 						<span class="footer_heading" id="connect_links"><%= t('shared.footer.connect') %></span>
 						<%= link_to(get_icon_text('envelope', t('common.form.email')), 'mailto:join.ifme@gmail.com') %>
+						<%= link_to(get_icon_text('rss', t('navigation.rss')), "https://medium.com/feed/ifme", target: "blank") %>
```

Comment:
Mind putting this in alphabet order, so before the Twitter link? :)

Reply:
oh, sure!

Reply:
Oh sorry, I meant in this html.erb file!

Reply:
oh, no need to apologize. My bad, I didn't look at the line to which you referred. I'll change that right now.